2017-bae-europe/france_only/france_only.ipynb

```diff
@@ -83,17 +78,15 @@
       "          iCCCCCLCf                                                           \n",
       "           .  C. ,                                                            \n",
       "              :                                                               \n",
-      "Experimental Warning: MatlSellPolicy is experimental and its API may be subject to change\n",
-      "Experimental Warning: the Mixer archetype is experimental\n",
       "\n",
       "Status: Cyclus run successful!\n",
       "Output location: cyclus.sqlite\n",
-      "Simulation ID: c68feb27-c186-44c9-b377-a4c98bc08bd0\n"
+      "Simulation ID: bb22bbdf-24b1-427f-bea9-482e8bd9988c\n"
      ]
     }
    ],
    "source": [
-    "! rm ./Desktop/cylus.sqlite\n",
+    "! rm ./cyclus.sqlite\n",
```

Comment:
The notebook's expected output name and the output name used in the script are different.
>The output file should be named
>
>      astrid.sqlite

Reply:
I didn't specify the output file name in this block, so the output file would be `cyclus.sqlite`.

Reply:
I realize that I made the mistake on the lower block. sorry! You're right.